src/TicketingSystem.EventsApi/Program.cs

```diff
@@ -0,0 +1,86 @@
+using AutoMapper;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using System;
+using System.IO;
+using System.Reflection;
+using TicketingSystem.BusinessLogic;
+using TicketingSystem.BusinessLogic.Mapper;
+
+namespace TicketingSystem.EventsApi
+{
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            var env = Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT")
+                ?? throw new ArgumentException("Missing ASPNETCORE_ENVIRONMENT env var");
+
+            var config = SetupConfiguration(env);
+            builder.Services.AddSingleton<IConfiguration>(config);
+
+            var connectionString = config.GetConnectionString("connectionString");
+            var databaseName = config.GetSection("databaseName").Value;
+
+            builder.Services.AddBusinessLogicServices(connectionString, databaseName);
```

Comment:
From a reader perspective this naming is confusing - looks like BusinessLogic depends on DB Name and connectionString. But you can leave it as is - I'm ok with the implementation of this extention method in general.

Reply:
As the chosen architectural approach assumes specific dependency chain (DAL <- BLL <- PL), there is a chain of calls of dependency injection registrations

It's possible to change the name to a more clear one which will reflect internal DAL dependency injection calls